packages/shared/createEventResponder.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ * @flow
+ */
+
+import type {ReactEventResponder} from 'shared/ReactTypes';
+import {REACT_RESPONDER_TYPE} from 'shared/ReactSymbols';
+
+export default function createEventResponder<E, C>(
+  displayName: string,
```

Comment:
Why is this a separate object instead of part of the config?

Reply:
So we ensure the hidden class is consistent, this improved perf in my microbenchmarks by around 15%.

Reply:
Do we need the `displayName` in prod?

(Regardless, doesn't block this PR)